src/Core/Search.php

```diff
@@ -240,7 +240,9 @@ public static function searchContact(string $search, string $mode, int $page = 1
 					$return = array_map(function ($result) {
 						static $contactType = [
 							'People'       => Contact::TYPE_PERSON,
+							// Kept for backward compatibility
 							'Forum'        => Contact::TYPE_COMMUNITY,
```

Comment:
This can be removed safely. The variable `$contactType` is only used at a single location, some lines below.

Reply:
What about older directory versions issuing this value in their results?

Reply:
Oh, I misread the code ...